code/game/jobs/job/civilian.dm

```diff
@@ -18,14 +18,11 @@
 		if(4) H.equip_or_collect(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
 	H.equip_or_collect(new /obj/item/clothing/under/color/random(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
-	if(H.backbag == 1)
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H), slot_r_hand)
-	else
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H.back), slot_in_backpack)

```

Comment:
Yes, I know, I just don't get why you left some `H.back`s in, and axed others =p 

If you're going to kill some off..kill them all off, please.